src/meshapi/models/node.py

```diff
@@ -26,6 +34,12 @@ class NodeStatus(models.TextChoices):
 
     status = models.CharField(choices=NodeStatus.choices, help_text="The current status of this Node")
 
+    map_display = models.CharField(
```

Comment:
Perhaps this could be named a little more generically, as this might be useful in other contexts besides the map. Perhaps we could name it something like `variant`, `type`, or `icon`?

Reply:
`display_variant`, even 😛

Reply:
`type` sounds good to me